test/common/http/http1/conn_pool_test.cc

```diff
@@ -269,13 +281,13 @@ TEST_F(Http1ConnPoolImplTest, VerifyBufferLimits) {
  * Tests a request that generates a new connection, completes, and then a second request that uses
  * the same connection.
  */
-TEST_F(Http1ConnPoolImplTest, MultipleRequestAndResponse) {
+TEST_F(Http1ConnPoolImplTest, Http10MultipleRequestAndResponse) {
```

Comment:
I think it would be better if you defaulted protocol in the codec to 1.1, and left all the existing tests alone, and then added some new 1.0 tests.

Reply:
Done.